inst/private/python_header.py

```diff
@@ -177,7 +177,7 @@ def octoutput(x, et):
             c = ET.SubElement(et, "list")
             for y in x:
                 octoutput(y, c)
-        elif isinstance(x, int):
+        elif isinstance(x, int) or (sys.version_info < (3, 0) and isinstance(x, long)):
```

Comment:
somewhat, just following the style of the local code, filed #775 to clean it up.